examples/panic/panic.bal

```diff
@@ -0,0 +1,31 @@
+import ballerina/io;
+
+type Record record {
+    int id;
+    string name;
+};
+
+// This example demonstrates panic an error. The next example demonstrates how to catch the panicked errors.
```

Comment:
What do we mean by "panic an error" here?

Also we shouldn't refer to the next example as that might not be straightforward navigation. It's better if we talk only about this example here and give a separate intro to the other example.

Reply:
I think we shouldn't be referring to as "panicking an error". See https://gobyexample.com/panic for example